httpclient5/src/main/java/org/apache/hc/client5/http/auth/NTCredentials.java

```diff
@@ -85,25 +89,46 @@ public NTCredentials(
      * Essentially, the computer name for this machine.
      * @param domain The domain to authenticate within.
      * @param netbiosDomain The netbios version of the domain name.
+     @deprecated (5.3) Use {@link #NTCredentials(char[], String, String, String)}
      */
+    @Deprecated
     public NTCredentials(
             final String userName,
             final char[] password,
             final String workstation,
             final String domain,
             final String netbiosDomain) {
+        this(password, userName,  domain, netbiosDomain);
+    }
+
+    /**
+     * Constructor to create an instance of NTCredentials.
+     *
+     * @param password      The password to use for authentication. Must not be null.
+     * @param userName      The user name for authentication. This should not include the domain to authenticate with.
+     *                      For example: "user" is correct whereas "DOMAIN&#x5c;user" is not. Must not be null.
+     * @param domain        The domain to authenticate within. Can be null.
+     * @param netbiosDomain The netbios version of the domain name. Can be null.
```

Comment:
I am confused about this description. Is it now the NetBIOS version or the domain?

Reply:
I'm confuse here, it's exactly the same that the other class.

`    * @param netbiosDomain The netbios version of the domain name.`

Reply:
Yes, but there is no correlation to the NetBIOS *domain*.

Reply:
changed.